drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -29,9 +30,8 @@ local COLOR_TEMPERATURE_KELVIN_MIN = 1
 local COLOR_TEMPERATURE_MIRED_MAX = CONVERSION_CONSTANT/COLOR_TEMPERATURE_KELVIN_MIN
 local COLOR_TEMPERATURE_MIRED_MIN = CONVERSION_CONSTANT/COLOR_TEMPERATURE_KELVIN_MAX
 
-local COMPONENT_TO_ENDPOINT_MAP = "__component_to_endpoint_map"
+local SWITCH_INITILIZED = "__switch_intialized"
```

Comment:
typo: INITIALIZED

Reply:
Done in ecc24c0b70e61b0b1ea56a8550c315aaf23e9158